pubsub/streaming-analytics/README.md

```diff
@@ -102,6 +103,7 @@ mvn compile exec:java \
   -Dexec.cleanupDaemonThreads=false \
   -Dexec.args="\
     --project=$PROJECT_NAME \
+    --region=us-central1 \
```

Comment:
what are your thoughts on instead of hardcoding a value here, make it `$REGION`? Since you are linking to the list of regions, it may be easier to allow the user to choose from there?

Reply:
Good idea! Fixed.